app/src/components/title.tsx

```diff
@@ -0,0 +1,28 @@
+export default function Title({
+  text,
+  dark = true,
+  textSize = "text-2xl",
+}: {
+  text: string;
+  dark?: boolean;
+  textSize?: string;
+}) {
+  var linesClassNames = "w-10 h-0.5 " + (dark ? "bg-blue-950" : "bg-white");
+
+  return (
+    <div className="flex items-center">
+      <div className={linesClassNames} />
```

Comment:
Info: what does this `div` do? I see that they are both before and after the h1.

Reply:
Only for visual, as in the figma component for titles.
![image](https://github.com/clicepfl/clic-website-v2/assets/56764430/a12840e5-6023-422c-a099-bce5c31673df)